components/Header/Header.tsx

```diff
@@ -17,14 +17,16 @@ export function Header(props: Props) {
   const toggleMenu = (pref: boolean) => setIsMenuOpen(pref);
 
   return (
-    <header
-      className=" sticky top-0 z-[50] flex h-[50px] items-center justify-between bg-[#252D54] md:h-[85px]"
-      {...props}
-    >
-      <Logo toggleMenu={toggleMenu} />
-      <BurgerMenu toggleMenu={toggleMenu} isMenuOpen={isMenuOpen} />
-      <NavigationAlt toggleMenu={toggleMenu} isMenuOpen={isMenuOpen} />
-      <Navigation />
-    </header>
+    <div className="bg-[#252D54]">
+      <header
+        className=" sticky top-0 z-[50] mx-auto flex h-[50px] max-w-[1600px] items-center justify-between bg-[#252D54] md:h-[85px]"
+        {...props}
```

Comment:
Don't recommend setting a hard height. What happens if I have more items in the header?
Also probably don't need the background here as well since the parent div already has it.